odcockpit/src/plugins/environmentviewer/EnvironmentViewerGLWidget.cpp

```diff
@@ -522,4 +520,4 @@ namespace cockpit {
             }
         }
     }
-} // plugins::environmentviewer
+} // plugins::environmentviewer銏聺籲籼x腻究|蚀硳}穷z要硵y誀籼z趇癴w誃
```

Comment:
Sorry for neglecting this strange line of code. I reverted the changes for both the h and cpp files. I also removed some duplicated lines which exit in EnvironmentViewerGLWidget.cpp in the master branch such as "using namespace odcore::data;"